inc/wpseo-functions.php

```diff
@@ -259,3 +259,50 @@ function wpseo_get_capabilities() {
 	}
 	return WPSEO_Capability_Manager_Factory::get()->get_capabilities();
 }
+
+
+/**
+ * Callback function to pass to the oEmbed's response data that will enable
+ * support for using the image and title set by the WordPress SEO plugin's fields. This
+ * address the concern where some social channels/subscribed use oEmebed data over OpenGraph data
+ * if both are present.
+ * 
+ * @param {array}  $data   - The oEmbed data
+ * @param {object} $post   - The current Post object
+ * @param {number} $width  - The current post's width (more applicable for attachments)
+ * @param {number} $height - The current post's height (more applicable for attachments)
+ *
+ * @see https://developer.wordpress.org/reference/hooks/oembed_response_data/ for hook info
+ *
+ * @return {array} an oEmbed array of data with modified values where appropriate.
+ */
+function wpseo_hook_oembed( $data, $post, $width, $height){
+  // data to be returned
```

Comment:
Can you add a full stop behind this comment.

Reply:
All set